index.html

```diff
@@ -198,15 +198,15 @@ <h1>Tonic</h1>
   <section id="about_me">
     <div class="row">
       <div class="about-column">
-        <h1>About<br> Myself</h1>
+        <h2>About<br> Myself</h2>
         <p id="presentation">Hello I’m a software developer! I can help you build a product ,
           feature or website Look through some of my work and experience!
           If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
         <div id="profile_link">
           <h2>LET’S CONNECT</h2>
           <ul>
             <li><a href="#"><img src="./images/github.svg" alt="Github"></a></li>
-            <li><a href="https://www.linkedin.com/in/athaliah-li-56217324a/"><img src="./images/Linkedin icon.svg"
+            <li><a href="https://www.linkedin.com/in/athaliah-li-56217324a/"><img src="./images/linkdin.svg"
                   alt="linkedin "></a></li>
             <li><a href="#"><img src="./images/angellist.svg" alt="angellist"></a></li>
             <li><a href="https://twitter.com/AthaliahLi"><img src="./images/twitter.svg" alt="twitter "></a></li>
```

Comment:
- [Optional] it seems that your text displaced during editing your code kindly fix that.
![conttttt](https://user-images.githubusercontent.com/66992901/201947396-2789a963-7015-4aa4-adca-1ace088fd73e.PNG)